src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -53,13 +57,15 @@ public Lift(RobotMap.LiftMap map) {
 
     // sets the ratchet to either be activated or deactivated depending on liftSpeed
     public void liftSpeed(double speed) {
-        if (speed > 0) {
-            elevatorBrake.set(false);
-            if (upperLimitSwitch.getAsBoolean()) {
+        if (Math.abs(speed) <= .1) {
+            speed = 0.0;
+        }
+        if (speed < 0) {
+            if (upperLimitSwitch.getAsBoolean() || !elevatorBrake.get()) {
                 speed = 0;
             }
-        } else if (speed <= 0) {
-            elevatorBrake.set(true);
+        } else if (speed > 0) {
```

Comment:
We need to be careful with the `>` and `<` here as they will likely change when we move to the triggers. The motion convention should be made clear in a comment at the top. (Positive input lowers the lift, negative raises it)

Reply:
Outdated We fixed this in Robot using the inverse of the Trigger values